tests/phpunit/elementor/modules/kit-elements-defaults/import-export/runners/mock/kit-elements-defaults.json

```diff
@@ -0,0 +1,17 @@
+{
+	"mock-widget": {
+		"text": "<script>Test</script> value",
+		"invalid_control": "some value",
+		"removed_on_import_and_on_export_method": "some value",
+		"__globals__": {
+			"color": "global-color",
+			"invalid_control2": "some value"
+		},
+		"__dynamic__": {
+			"invalid_control3": "some value"
+		}
```

Comment:
do you wanna add a valid control with some HTML in it to make sure it's sanitized?

Reply:
Done